internal/servers/worker/controller_connection.go

```diff
@@ -34,7 +36,6 @@ func (w *Worker) startControllerConnections() error {
 		default:
 			host, port, err := net.SplitHostPort(addr)
 			if err != nil && strings.Contains(err.Error(), "missing port in address") {
-				w.logger.Trace("missing port in controller address, using port 9201", "address", addr)
```

Comment:
Did you choose to elide this instead of convert it on purpose? Not saying it _needs_ to be something published but also not sure if it was just an oversight.

Reply:
It was intentional and part of commit 3c1a3bd